app/views/application/_head.html.erb

```diff
@@ -1,6 +1,7 @@
 <head>
   <title><%= title %></title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <%= yield :meta_description %>
```

Comment:
Does this need an `if content_for? :meta_description`? I actually have no idea what happens if you yield to something that isn't defined.